package.json

```diff
@@ -249,9 +248,7 @@
         "mousetrap": "^1.6.3",
         "msw": "^2.1.5",
         "msw-storybook-addon": "^2.0.2",
-        "mutationobserver-shim": "^0.3.3",
-        "npm-run-all": "^4.1.3",
-        "nsp": "^3.2.1",
```

Comment:
package is deprecated and service is down. i dont see where it's used?
https://www.npmjs.com/package/nsp